src/content_sao.cpp

```diff
@@ -1325,6 +1325,15 @@ u16 PlayerSAO::punch(v3f dir,
 	return hitparams.wear;
 }
 
+void PlayerSAO::rightClick(ServerActiveObject *clicker)
+{
+	PlayerSAO *playersao = m_player->getPlayerSAO();
+
+	if (isAttached())
```

Comment:
Should also be allowed for attached players. You can punch them, so why not right-click?